libraries/AP_HAL_ChibiOS/hwdef/BETAFPV-F405-I2C/hwdef.dat

```diff
@@ -3,7 +3,7 @@
 include ../BETAFPV-F405/hwdef.dat
 
 # undefine the pins we are going to change
-undef PB4 PB10 PB11 PC6 PA3 PA2
+undef PB4 PC6 PB10 PB11 PA3 PA2
```

Comment:
the undef of PC6 just re-adds PC6 with same line?

Reply:
Fixed